controls/e8.yml

```diff
@@ -0,0 +1,177 @@
+id: 'e8'
+policy: 'e8'
+source: 'https://www.cyber.gov.au/sites/default/files/2023-11/PROTECT%20-%20Hardening%20Linux%20Workstations%20and%20Servers%20%28November%202023%29.pdf'
+title: 'Australian Cyber Security Centre (ACSC)'
+levels:
+    - id: base
+
+controls:
+    -   id: 'patching'
+        levels:
+            - base
+        title: 'Application and operating system patching'
+        rules:
+            - package_talk_removed
+            - package_talk-server_removed
```

Comment:
Packages like talk, ypbind, quagga, squid aren't part of RHEL 10 so I think these rules should be excluded in the RHEL 10 profile.